### PR TITLE
Use h1 in bundle version

### LINKED ISSUE #831: Use `h1` in `bundle version`

### PR DESCRIPTION
**Pull request by tnir**
Closes #831

Part of #830

Signed-off-by: Takuya Noguchi [takninnovationresearch@gmail.com](https://github.com/sponsors/tnir)

### CODE REVIEW

**Comment by tnir** [comment]
@deivid-rodriguez or any other maintainer(s) Deploying to review app is failing. Could you take a look and/or just put logs here?

**Comment by deivid-rodriguez** [comment]
```
-----> Building on the Heroku-22 stack
-----> Using buildpacks:
       1. heroku/nodejs
       2. heroku/ruby
-----> Node.js app detected
       
-----> Creating runtime environment
       
       NPM_CONFIG_PRODUCTION=false
       NPM_CONFIG_LOGLEVEL=error
       NODE_VERBOSE=false
       NODE_ENV=production
       NODE_MODULES_CACHE=true
       
-----> Installing binaries
       engines.node (package.json):  unspecified
       engines.npm (package.json):   unspecified (use default)
       
       Resolving node version 16.x...
       Downloading and installing node 16.16.0...
       Using default npm version: 8.11.0
       
-----> Installing dependencies
       Installing node modules
       npm ERR! code EEXIST
       npm ERR! syscall open
       npm ERR! path /tmp/npmcache.Crc5g/_cacache/tmp/06c75274
       npm ERR! errno -17
       npm ERR! EEXIST: file already exists, open '/tmp/npmcache.Crc5g/_cacache/tmp/06c75274'
       npm ERR! File exists: /tmp/npmcache.Crc5g/_cacache/tmp/06c75274
       npm ERR! Remove the existing file and try again, or run npm
       npm ERR! with --force to overwrite files recklessly.
       
       npm ERR! A complete log of this run can be found in:
       npm ERR!     /tmp/npmcache.Crc5g/_logs/2022-08-01T10_21_41_558Z-debug-0.log
-----> Build failed
       
       We're sorry this build is failing! You can troubleshoot common issues here:
       https://devcenter.heroku.com/articles/troubleshooting-node-deploys
       
       Some possible problems:
       
       - Node version not specified in package.json
         https://devcenter.heroku.com/articles/nodejs-support#specifying-a-node-js-version
       
       Love,
       Heroku
       
 !     Push rejected, failed to compile Node.js app.
 !     Push failed
```

**Comment by tnir** [comment]
Thanks. It looks like a problem on Heroku side, and let me re-push the branch to trigger the app.

File: source/v1.13/bundle_version.html.haml [hunk 1]
```diff
@@ -1,4 +1,8 @@
-%h2 bundle version
+---
+title: bundle version
+description: Prints the bundler's version information
```
Comment (olleolleolle) [inline source/v1.13/bundle_version.html.haml:3]:
```suggestion
description: Prints Bundler's version information
```

How about this? Or, does "the bundler" convey anything else than "Bundler itself"?

Comment (tnir) [reply source/v1.13/bundle_version.html.haml:3]: Actually I just copied from the following v1.12 content (which however I forgot to add it to this commit 😜 ):

https://github.com/rubygems/bundler-site/blob/83b575b57c143d9a5bdc935ab8c0269719a2335a/source/v1.12/bundle_version.html.md?plain=1#L3

I think this explanation should be consistent with what Bundler CLI shows with `bundle help version`:
https://github.com/rubygems/rubygems/blob/ca4e6cd72d08cc1320f5931b9fcc1175d5e6e279/bundler/spec/commands/help_spec.rb#L25-L28

**Review by olleolleolle — APPROVED** [review]
Looks good!

Thanks for giving context for changes, it helps so much!

**Pull request merged**